<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>portes</class>
 <widget class="QDialog" name="portes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_port">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>471</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>NUM</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_NUMPORT">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_CODEPORT">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Ajouter">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>AJOUTER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Afficher">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TRI">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tri</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Recherche : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_search">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>901</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/back/images.png);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>130</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CODE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>190</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TEMP OUVERT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>230</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TEMP FERME</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_TEMPOUVERT">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>190</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_TEMPFERME">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>230</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>460</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>0</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Numero</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Code</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>tableView_port</zorder>
  <zorder>label</zorder>
  <zorder>spinBox_NUMPORT</zorder>
  <zorder>spinBox_CODEPORT</zorder>
  <zorder>pushButton_Ajouter</zorder>
  <zorder>pushButton_Afficher</zorder>
  <zorder>pushButton_TRI</zorder>
  <zorder>label_5</zorder>
  <zorder>txt_search</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
